internal/controllers/cluster/cluster_controller.go

```diff
@@ -179,7 +179,7 @@ func patchCluster(ctx context.Context, patchHelper *patch.Helper, cluster *clust
 
 // reconcile handles cluster reconciliation.
 func (r *Reconciler) reconcile(ctx context.Context, cluster *clusterv1.Cluster) (ctrl.Result, error) {
-	log := ctrl.LoggerFrom(ctx, "cluster", cluster.Name)
+	log := ctrl.LoggerFrom(ctx)
```

Comment:
Are we losing the k/v here?

Reply:
Should we add the Cluster k/v in the `Reconcile` func?

Reply:
I don't think so.